app/assets/stylesheets/searchworks4/search-results.css

```diff
@@ -164,3 +164,25 @@ details[open] {
     -webkit-text-stroke: 0.45px;
   }
 }
+
+.collection-info {
+  background-color: var(--stanford-fog-light);
+
+  .finding-aid a {
+    --bs-link-decoration: none;
+
+    &::after {
+      content: '\F144';
```

Comment:
Is there a reason not to use the `<i class="bi bi-arrow-up-right"></i>` in the markup?  I think that is much easier to find and follows along with how we are doing most of the other icons.

Reply:
you mean in collection.marc_links.finding_aid.first.html.html_safe? How?